DataFormats/Streamer/src/classes_def.xml

```diff
@@ -1,6 +1,8 @@
 
 <lcgdict>
- <class name="edm::StreamedProduct" ClassVersion="12">
+ <class name="edm::StreamedProduct" ClassVersion="14">
+  <version ClassVersion="14" checksum="855998814"/>
+  <version ClassVersion="13" checksum="87489871"/>
   <version ClassVersion="12" checksum="855998814"/>
```

Comment:
Hi,  I thought I closed the PR 9814 quite a while ago.
________________________________
From: Chris Jones [notifications@github.com]
Sent: 21 July 2015 16:29
To: cms-sw/cmssw
Cc: Sunanda Banerjee
Subject: Re: [cmssw] bsunanda:Run2 alca16 (#9814)


In DataFormats/Streamer/src/classes_def.xml<https://github.com/cms-sw/cmssw/pull/9814#discussion_r35107808>:

> @@ -1,6 +1,8 @@
>
>  <lcgdict>
> - <class name="edm::StreamedProduct" ClassVersion="12">
> + <class name="edm::StreamedProduct" ClassVersion="14">
> +  <version ClassVersion="14" checksum="855998814"/>
> +  <version ClassVersion="13" checksum="87489871"/>
>    <version ClassVersion="12" checksum="855998814"/>


Why was the checksums changed? The checksum for 12 and 14 are identical.

—
Reply to this email directly or view it on GitHub<https://github.com/cms-sw/cmssw/pull/9814/files#r35107808>.